GooglePlayInstant/Tests/Editor/QuickDeploy/HttpRequestHelperTestHelper.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+
+namespace GooglePlayInstant.Tests.Editor.QuickDeploy
+{
+    /// <summary>
+    /// Contains helper methods for HttpRequestHelperTest class.
+    /// </summary>
+    public static class HttpRequestHelperTestHelper
```

Comment:
After you've made the suggested changes above, if you still need any helper methods pleas move them into the HttpRequestHelperTest class and make them private and delete this file.

Reply:
Resolved.